### DISCUSSION TITLE
v19.0.0.0beta2

### DISCUSSION THREAD

**Discussion by awawa-dev** [awawa-dev/HyperHDR#458] [General]
Link to the release: https://github.com/awawa-dev/HyperHDR/releases/tag/v19.0.0.0beta2

I think that the number of changes and fixes introduced recently fully justifies the release of one more test version before the stable version.

New features:
- HyperHDR got new packages repository (Github Pages). More details: https://awawa-dev.github.io/ Also RPM packages are available there and Arch Linux installers are planned.
- The SD images that come with this release are already linked to the HyperHDR repository, so the future update should be much simpler.
- Add support for ESP32-S2 mini board. It requires latest HyperSerialESP32 v8 firmware https://github.com/awawa-dev/HyperSerialESP32/releases/tag/v8.0.0.0 . This firmware brings also support for multi-segments, that allows to double the refresh rate of large sk6812/ws2812b setups for free (well, at least proper LED strip design is required (#432) 
- Save/restore WLED state and set max brightness at startup (#353) 
- Add support for utv007 / Linux (#423) 

 Fix:
- Automatic signal detection timing (#410) 
- Fix the black color for disabled LEDs (#419) 

Improvements:
- Improved LED device performance (dropping frames that have exceeded the hardware capabilities of the LED, UDP/SPI devices are not affected as the process is handled by the OS)
- Update de.json (#411) Thanks @Duese123
- Update Github actions/upload@v3 (#373)

**Comment by Bascht74** [comment]
Great new release! 
I switched from Version 14 with warm white LEDs 60 per meter to V19Beta2 and cold white LEDs with 96 per meter. Much, much better!!! More accurate colors after using the right LUT.

I have a some questions left: 
Can I solve a problem in my setup with the new disable and delete function?
I have a 72" TV and a 5m strip. That is a little too short, so the stripes on the left and right side a not long enough at the bottom. There a 8 LEDs missing.  I need to add them in the LED layout and then delete the LEDs at the position they are missing.  Is that possible?
I am not sure how disable and delete works internally. What I understood is: 
1. If you disable a LED it is still counted but it will be kept black. I could use that if one LED is broken (e.g. stroking when brightened) and everything works if that LED is black
2. If you delete a LED it won't be counted. I could physically remove the broken LED completely (connect the signal to the next one) and I can later delete this LED within HyperHDR on its position. So it will be skipped, but the other (not deleted) LEDs are keeping there current position. So the geometry is fine

Is that right?

BTW. 
I don't get the use case for moving a LED? 
What Do you have in mind for that? EG. use other positions (e.g. middle of the screen) to light other ambient lights? Are there more?

Seb

**Comment by awawa-dev** [comment]
Hi
> Great new release!
I switched from Version 14 with warm white LEDs 60 per meter to V19Beta2 and cold white LEDs with 96 per meter. Much, much better!!! More accurate colors after using the right LUT.

Thank you :)

> I have a 72" TV and a 5m strip. That is a little too short, so the stripes on the left and right side a not long enough at the bottom. There a 8 LEDs missing. I need to add them in the LED layout and then delete the LEDs at the position they are missing. Is that possible?

Yes, it's possible just like you said. Just build the LED strip like there were these 8 LEDs present and later click on each and delete it. I don't know where the gap is, and maybe it will be necessary to relocate the remaining LEDs into proper locations at the bottom (The 'move' function may come in handy).

> I don't get the use case for moving a LED?

Not every LED instance setup is based on rectangle. For example Philips Hue lamps, you can precise locate the area for them.

**Comment by carlks** [comment]
Hi awawa
Looking to build a fresh setup for a 120" projector screen. What is the current recommended setup since this new release?

Thanks
Carl

**Reply by awawa-dev** [reply]
you need to count how many LEds will come out for the whole setup. Then decide if the sk6812 RGBW CW refresh rate using ESP32 and HyperSerialESP32 with multi-segment mode is satisfactory for you. Sample results are provided on the project page (scaling is linear). If not, use SPI LEDs which will provide better refreshrate but much worse white and color shift mostly towards blue.

**Comment by thecourteener** [comment]
awawa, where do I find HyperHDR v19 beta 2 SD card image for a Raspberry Pi 4? I can only see SD-card-image-32bits-armv6l and 
SD-card-image-64bits-aarch64

**Reply by awawa-dev** [reply]
Raspberry Pi OS lite is offered only in two versions: arm6l and aarch64 anyway. HyperHDR arm7l SD image was introduced artificially: it was arm6l OS and HyperHDR compiled for arm7l. I dropped it since the HyperHDR repo was introduced in v19beta2: manual is not yet updated. You can use both images although aarch64 is recommended for Rpi4.
